index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
             </ul>
         </div>
       </section>
+      <section class="works">
+        <div class="recent-works">
+          <h2 id="worksTitle">My Recent Works</h2>  
+        </div>
+        <div id="grid-container">
+            <div class="recent-project">
+              <img class="project-img" src="img/Img-Placeholder.png" alt="img"/>
+              <div class="right-block">
+                 <h2 class="title-post">Multi-Post Stories</h2>
+                 <p class="posts-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                 <ul class="tech-skills">
+                  <li class="skillset">CSS</li>
+                  <li class="skillset">HTML</li>
+                  <li class="skillset">Bootstrap</li>
+                  <li class="skillset">Ruby</li>
+                 </ul>
+                 <button class="project-view"><p>See project</p></button>
+                </div>
+            </div>
+               <!--flexbox individual cards-->
+            <div class="cards-box">
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
```

Comment:
- [ ] Please stick to the design as much as possible. The card is at the center of the page with equal spacing please update it.
- [ ] The list HTML, Bootstrap, and Ruby has different colors please update it
- [ ] Also See buttons have different color and has no black border please stick to the design. I would suggest please increase the height and give more padding for buttons so It matches the design.
- [ ] I would suggest you please update with all the cards
### Your design

![his](https://user-images.githubusercontent.com/81727988/236487528-2803f9dc-48ba-494f-9e3f-96168dd2bea8.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236487571-afb2d921-2376-4de4-970d-83714af53d8b.png)